script/WormholeL2FlexibleVotingDeploy.s.sol

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: MIT
```

Comment:
*arbitrary comment location*

We should update the README to reflect the changes to the scripts. We should also include some examples on how to use them, especially the main deploy script, and include explanations for what should/shouldn't be set in environment variables for a test deployment vs. a real deployment.

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/68/commits/b893d4131064c6e75884eb0323a874fe0d599e1a